Gemfile.lock

```diff
@@ -41,16 +51,22 @@ GEM
     simplecov-html (0.11.0)
     terminal-table (1.8.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
+    twilio-ruby (5.41.0)
+      faraday (>= 0.9, < 2.0)
+      jwt (>= 1.5, <= 2.5)
+      nokogiri (>= 1.6, < 2.0)
     unicode-display_width (1.6.1)
 
 PLATFORMS
   ruby
 
 DEPENDENCIES
+  colorize
   rspec
   rubocop (= 0.79.0)
   simplecov
   simplecov-console
+  twilio-ruby
 
 BUNDLED WITH
    2.1.4
```

Comment:
Nicely done, updating the whole Gemfile.lock. I did not do it, probably should have.